### PR TITLE
fix: revert old style

### PR DESCRIPTION
**Pull request by cbourgois**
### Description of the Change

b2135fe — fix: revert old style

### TODO : 
- [x] fix virtual scroll

### CODE REVIEW

**Comment by antleblanc** [comment]
As `angular-vs-repeat` is now removed from the dependency list, the `TODO` list could be also updated/removed right?

File: package.json [hunk 1]
```diff
@@ -40,16 +40,16 @@
     "husky": "^1.1.0",
     "lint-staged": "^7.3.0",
     "npm-run-all": "^4.1.5",
+    "ovh-ui-kit": "^2.21.5",
```
Comment (antleblanc) [inline package.json]: Latest 2.x version could be specified (`^2.32.0`).

File: src/index.less [hunk 2]
```diff
@@ -1,443 +1,453 @@
...
-// @SIDEBAR_MENU_LVL2_ICON_FONT_SIZE        : 18px;
-// @SIDEBAR_MENU_LVL3_ICON_FONT_SIZE        : 14px;
-
-// // ACTIONS BUTTON
-@SIDEBAR_MENU_ACTIONS_MENU_BG            : #00a2bf;
-// @SIDEBAR_MENU_ACTIONS_MENU_COLOR         : @SIDEBAR_MENU_COLOR;
-// @SIDEBAR_MENU_ACTIONS_MENU_OPEN_COLOR    : @SIDEBAR_MENU_TEXT_ACTIVE_COLOR;
-
-// /* Hide for mobile, show later */
+@import "../node_modules/ovh-ui-kit/packages/oui-responsive/_variables";
```
Comment (antleblanc) [inline src/index.less]: `~` shorthand couldn't be used? (ref: [rollup-plugin-less-tilde-importer](https://github.com/ovh-ux/rollup-plugin-less-tilde-importer))

**Comment by ovh-cds** [comment]
CDS Report build#207.0 ✘  
* 
  * build ✔ 
  * test ✘

**Pull request merged**